app/models/campaign.rb

```diff
@@ -0,0 +1,15 @@
+class Campaign < ApplicationRecord
+  belongs_to :user
+  validates :title, presence: true,
+                    uniqueness: true,
+                    length: { in: 3..30 },
+                    format: { with: /\A[a-zA-Z0-9\s]+\z/ }
+  validates :description, presence: true,
+                          length: { maximum: 2000 }
+  validates :current_amount, :needed_amount, presence: true,
+                                             numericality: true,
+                                             on: :create
+  validates :requisite, presence: true,
+                        format: { with: /\A[+]?\d+\z/ },
```

Comment:
explain these validations

Reply:
To let enter numbers only as it's a string format